config/app.default.php

```diff
@@ -236,7 +236,7 @@
             'timezone' => 'UTC',
             'flags' => [],
             'cacheMetadata' => true,
-            'log' => false,
+            'log' => true,
```

Comment:
Wouldn't people be using debugkit out of the box which includes/enables query logging?